schemas/iio/adc.yaml

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: GPL-2.0
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/iio/adc.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: Generic IIO bindings for ADC channels
+
+select: true
```

Comment:
Better to keep this on in the kernel tree perhaps as files will need to reference it anyway?

I didn't think this one through as obvious it will blow up..